charts/node-analyzer/templates/configmap-image-analyzer.yaml

```diff
@@ -0,0 +1,94 @@
+{{ if not .Values.secure.vulnerabilityManagement.newEngineOnly }}
+{{- if include "deploy-nia" . }}
```

Comment:
This configmap is basically two configmaps (one for NIA, one for NA) rolled in to one. We should probably remove everything related to the NIA from this new chart.